lib/sdf-v1-routes-session/src/lib.rs

```diff
@@ -59,28 +55,14 @@ pub enum SessionError {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
-struct AuthApiErrBody {
+pub struct AuthApiErrBody {
     pub kind: String,
     pub message: String,
 }
 
 pub type SessionResult<T> = std::result::Result<T, SessionError>;
 
-impl IntoResponse for SessionError {
-    fn into_response(self) -> Response {
-        let (status_code, error_message) = match self {
-            Self::LoginFailed => (StatusCode::CONFLICT, self.to_string()),
-            Self::Workspace(WorkspaceError::WorkspaceNotFound(_)) => {
-                (StatusCode::CONFLICT, self.to_string())
-            }
-            Self::WorkspacePermission(_) => (StatusCode::UNAUTHORIZED, self.to_string()),
-            Self::AuthApiError(_) => (StatusCode::UNAUTHORIZED, self.to_string()),
-            _ => (StatusCode::INTERNAL_SERVER_ERROR, self.to_string()),
-        };
-
-        ApiError::new(status_code, error_message).into_response()
-    }
-}
+impl_default_error_into_response!(SessionError);
```

Comment:
We don't want to get rid of our custom error handling for SessionError

Reply:
fixed!